.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
```

Comment:
Like mentioned in https://github.com/wasmCloud/wasmCloud/pull/335, I'd expect "anything WASI" to be automatically provisioned by `rustup`, given that we have `rust-toolchain.toml` file

Reply:
The features run while the docker image is being built, before the repository is mounted - so they don't have access to that context... We could use a "postAttachCommand" to run rustup though - I need to test it, and ATM am having some issues getting any tests to compile (both from this branch and on upstream main, both within the containers and outside of them) and have other things I need to work on. I'll try again tomorrow.

Reply:
replaced with running `rustup show` in a post attach command. Note that originally I looked at other options like `rustup update` - but they didn't install the active toolchain & targets, while `show` did.